README.md

```diff
@@ -731,17 +727,19 @@ await directoryClient.deleteManifest();
 
 ### Import
 
+`createAsyncIterable` has been deprecated, please use `createImportRequest`
```

Comment:
it was not removed, right? (in other words the code for the rag-service still works?)

Reply:
Yes, still works, just deprecated